mdms/mdms-analytics/src/main/scala/de/hpi/isg/mdms/analytics/TableImportance.scala

```diff
@@ -0,0 +1,144 @@
+package de.hpi.isg.mdms.analytics
+
+import de.hpi.isg.mdms.domain.constraints.{ColumnStatistics, InclusionDependency, TupleCount}
+import de.hpi.isg.mdms.model.MetadataStore
+import de.hpi.isg.mdms.model.constraints.ConstraintCollection
+import de.hpi.isg.mdms.model.util.IdUtils
+import org.qcri.rheem.api.{DataQuanta, PlanBuilder}
+
+class TableImportance()(implicit planBuilder: PlanBuilder) {
+  // Calculating the probability matrix
+  def probMatrix(idUtils: IdUtils,
```

Comment:
Please add a Scaladoc. This is particularly crucial as this method is meant to be invoked by users.